modules/kubernetes/eks-core/k8s-cluster-role-binding.tf

```diff
@@ -37,3 +37,24 @@ resource "kubernetes_cluster_role_binding" "cluster_view" {
     api_group = "rbac.authorization.k8s.io"
   }
 }
+
+resource "kubernetes_cluster_role_binding" "get_node" {
```

Comment:
Maybe we want to use the same naming as aks. e.g `get_nodes`?

Reply:
Fixed